serving/samples/helloworld-clojure/README.md

```diff
@@ -75,9 +75,8 @@ recreate the source files from this folder.
    # Build an uberjar release artifact.
    RUN mv "$(lein uberjar | sed -n 's/^Created \(.*standalone\.jar\)/\1/p')" app-standalone.jar
 
-   # Configure and document the service HTTP port.
+   # Configure the service HTTP port.
```

Comment:
Can you update the ReadMes as well?

Reply:
Working on it. This seems like a good spot for automation.